src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/IncompleteServiceReportITest.java

```diff
@@ -0,0 +1,233 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.StandardReportResponse;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportControllerITest;
+import uk.gov.hmcts.juror.api.moj.report.ReportHashMap;
+import uk.gov.hmcts.juror.api.moj.report.ReportLinkedMap;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+@Sql({
+    "/db/mod/truncate.sql",
+    "/db/mod/reports/IncompleteServiceReportITest_typical.sql"
+})
+@SuppressWarnings({
+    "PMD.LawOfDemeter",
+    "PMD.JUnitTestsShouldIncludeAssert"//False positive
+})
+class IncompleteServiceReportITest extends AbstractStandardReportControllerITest {
+    @Autowired
+    public IncompleteServiceReportITest(TestRestTemplate template) {
+        super(template, IncompleteServiceReport.class);
+    }
+
+    @Override
+    protected String getValidJwt() {
+        return getCourtJwt("415");
+    }
+
+    @Override
+    protected StandardReportRequest getValidPayload() {
+        return addReportType(StandardReportRequest.builder()
+            .reportType("IncompleteServiceReport")
+            .date(LocalDate.now())
+            .locCode("415")
+            .build());
+    }
+
+    @Test
+    void positiveTypicalCourt() {
+        testBuilder()
+            .triggerValid()
+            .responseConsumer(this::verifyAndRemoveReportCreated)
+            .assertEquals(getTypicalResponse());
+    }
+
+    @Test
+    void negativeNoRecordsFound() {
+        StandardReportRequest request = getValidPayload();
+        request.setDate(LocalDate.now().minusDays(100));
```

Comment:
Can this be a fixed date? 

Else in 100 days time this will fail as the date will be after the sql date

Reply:
this should always work as the data in the sql is relative as well and I don't expect anything to come back thats 100 days old